### PR TITLE
Make Korath and Remnant cargo more lore accurate

### PR DESCRIPTION
**Pull request by warp-core**
**Bugfix:**  This PR addresses a request from @Zitchas on Discord.

## Fix Details
Korath bring outfits for the Remnant and take commodities.
Remnant bring commodities for the Korath and take outfits.
Korath hunting void sprites only carry void sprite parts.
Korath ships don't carry Remnant outfits.

### CODE REVIEW

File: data/korath/korath.txt [hunk 1]
```diff
@@ -852,4 +854,24 @@ outfitter "Korath Exiles"
 	"Outfits Expansion"
 	"Cargo Expansion"
 	"Local Map"
-	"Thermal Repeater Rifle"
+	"Thermal Repeater Rifle".
```
Comment (Amazinite) [inline data/korath/korath.txt:857]:
```suggestion
	"Thermal Repeater Rifle"
```

File: data/korath/korath.txt [hunk 2]
```diff
@@ -852,4 +854,24 @@ outfitter "Korath Exiles"
...
+	"Thruster (Comet Class)"
+	"Thruster (Planetary Class)"
+	"Generator (Furnace Class)"
+	"Triple Plasma Core"
+	"Small Heat Shunt"
+	"Large Heat Shunt"
+	"Fuel Processor"
+	"Systems Core (Small)"
+	"Systems Core (Medium)"
+	"Systems Core (Large)""
```
Comment (Amazinite) [inline data/korath/korath.txt:877]:
```suggestion
	"Systems Core (Large)"
```

**Review by Amazinite — CHANGES_REQUESTED** [review]
The single-word commodity names should probably be in quotes.

File: data/remnant/remnant 2 cognizance.txt [hunk 3]
```diff
@@ -447,7 +450,8 @@ mission "Remnant: Cognizance 12"
 		personality timid launching escort
 		fleet
 			names "remnant"
-			cargo 8 equipment
+			cargo 1
+			commodities Equipment
```
Comment (Amazinite) [inline data/remnant/remnant 2 cognizance.txt:454]:
```suggestion
			commodities "Equipment"
```

File: data/remnant/remnant.txt [hunk 4]
```diff
@@ -160,7 +160,7 @@ fleet "Remnant Home Guard"
 	names "remnant"
 	cargo 2
 	commodities Food Metal Plastic "Heavy Metals"
```
Comment (Amazinite) [inline data/remnant/remnant.txt:162]:
```suggestion
	commodities "Food" "Metal" "Plastic" "Heavy Metals"
```

**Review by Amazinite — APPROVED** [review]

**Comment by warp-core** [comment]
I do wonder if some of the bounty ships should perhaps have Remnant outfits?

**Pull request merged**